### PR TITLE
[KNOX-190]: Multiple Issue Comments and No Repeat Comments

### PR DESCRIPTION
**Pull request by alehechka**
## 📑 Description
- Adds the ability to supply a CSV list of Jira tickets to add a comment to.
- Adds the ability to check issues for existing comments and to skip adding a comment if it already exists. (Can be turned off via input)

> @dmsi-io/devops I plan to make a PR against the original repo after merging, but for the time being I'd like to be able to use this new functionality.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
     const result = await new Action({
       githubEvent,
       argv: parseArgs(),
       config,
-    }).execute()
+    }).execute();
 
     if (result) {
-      const extendedConfig = Object.assign({}, config, result)
+      const extendedConfig = Object.assign({}, config, result);
```
Comment (hbiede) [inline index.js:22]: This could just be an object spread

Comment (alehechka) [reply index.js:22]: yeah... but idk if I want to go around changing code that I don't need to, already bad enough that my vs code setup added semi colons everywhere

Comment (brianspowers) [reply index.js:22]: Ugh is this project in standard style or something like that?  Now I want to run down their eslint config...

Comment (alehechka) [reply index.js:22]: their eslint config is really terrible. it was red-underlining line endings (LF vs CRLF)

**Comment by alehechka** [comment]
Failing Gracefully: https://github.com/dmsi-io/gha-pull-request/runs/4595942369?check_suite_focus=true

Jira Comment Exists: https://github.com/dmsi-io/gha-pull-request/runs/4590386179?check_suite_focus=true

Adding Jira Comment: https://github.com/dmsi-io/gha-pull-request/runs/4590347275?check_suite_focus=true

**Review by brianspowers — APPROVED** [review]

**Review by hbiede — APPROVED** [review]

**Pull request merged**